BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -256,8 +261,83 @@ extension StoresTableViewController {
             }
             
         }
-    
+        
     }
+    
+}
 
+extension StoresTableViewController {
+    
+    func setsEdgePanGesture() {
+        
+        edgeGesture.addTarget(self, action: #selector(self.handleRightEdge(gesture:)))
+        edgeGesture.edges = .right
+        edgeGesture.delegate = self
+        
+        self.view.addGestureRecognizer(edgeGesture)
+        
+    }
+    
+    func handleRightEdge(gesture: UIScreenEdgePanGestureRecognizer) {
+        
+        switch gesture.state {
+            
+        case .began, .changed:
```

Comment:
`case` needs to be indented from `switch`, even though XCode does not do it for you.
That is a standard practice accross the industry.